packages/aws-cdk-lib/aws-stepfunctions/lib/states/succeed.ts

```diff
@@ -1,54 +1,49 @@
 import { Construct } from 'constructs';
 import { StateType } from './private/state-type';
-import { State } from './state';
-import { INextable } from '../types';
+import { JsonataCommonOptions, JsonPathCommonOptions, State, StateBaseProps } from './state';
+import { INextable, QueryLanguage } from '../types';
 
+interface SucceedBaseProps extends StateBaseProps {}
+interface SucceedJsonPathOptions extends JsonPathCommonOptions {}
+interface SucceedJsonataOptions extends JsonataCommonOptions {}
+/**
+ * Properties for defining a Succeed state that using JSONPath
+ */
+export interface SucceedJsonPathProps extends SucceedBaseProps, SucceedJsonPathOptions {}
+/**
+ * Properties for defining a Succeed state that using JSONata
+ */
+export interface SucceedJsonataProps extends SucceedBaseProps, SucceedJsonataOptions {}
```

Comment:
I know you're trying to make the type consistent bewteen different states and differentiate options vs props for future uses, but IMO this really make the readability harder with all the type inferences. 

Let me know your thought but I think we can just do:
```
export interface SucceedJsonPathProps extends StateBaseProps, JsonPathCommonOptions {}
```

This also applies to all the other states.